environments/production/public.yml

```diff
@@ -256,3 +256,5 @@ shared_dir_gid: 1500
 # for better time syncronization on aws servers
 # replace ntp with chrony.
 use_chrony: True
+
+ASYNC_INDICATORS_TO_QUEUE: 20000
```

Comment:
This needs to be part of the `localsettings` object for it to be included in the localsettings https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2#L882

Reply:
Ah, good catch. I've moved it under the localsettings object ([c8c14a8](https://github.com/dimagi/commcare-cloud/pull/6237/commits/c8c14a8a42f6420f5173fc0bf6559b9fe5d51338)).

Reply:
Thanks for catching that @calellowitz